service/debugger/debugger.go

```diff
@@ -757,6 +757,13 @@ func isBreakpointExistsErr(err error) bool {
 	return r
 }
 
+func (d *Debugger) CreateTracepoint(fnName string) error {
```

Comment:
I'm not sure about the name of this function, it seems confusing with the old meaning of Tracepoint. `CreateBPFTracepoint`, `CreateBufferedTracepoint`, `CreateAsyncTracepoint`, `CreateFastTracepoint`? Same for `SetTracepoint`.

Reply:
Done.

Reply:
I meant the name in the JSON-RPC API should also change.